addons/event_sale/data/event_sale_demo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="event_sale_order_pricelist_1" model="product.pricelist">
+            <field name="name">Euro</field>
+            <field name="currency_id" ref="base.USD"/>
+        </record>
+
+        <record id="event_sale_order_pricelist_2" model="product.pricelist">
+            <field name="name">Dollar</field>
+            <field name="currency_id" ref="base.EUR"/>
+        </record>
+
+        <record id="event_sale_order_1" model="sale.order">
```

Comment:
Could be great to add a bit more details, and link the SO to registrations already existing in event for example. That way demo is consistent.